packages/ciphernode/events/src/enclave_event/mod.rs

```diff
@@ -139,6 +139,11 @@ impl EnclaveEvent {
             _ => false,
         }
     }
+
+    pub fn event_type(&self) -> String {
```

Comment:
This looks duplicated below so I believe we can delete this.

Reply:
Good Catch!